src/Microsoft.ML.TimeSeries/STL/FastLoess.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections.Generic;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.TimeSeries
+{
+    /// <summary>
+    /// this is the fast version of Loess. there are several alternatives to improve the performance. this one is an approximation approach.
+    /// the smoothing is conducted on a sample set, and then the values on the left points are assigned directly.
+    /// </summary>
+    internal class FastLoess
+    {
+        /// <summary>
+        /// this class is a sampling based method, so here specifies the sample size.
```

Comment:
nit: Capitalize beginning of sentences.

Reply:
Updated.

---
In reply to: [442520500](https://github.com/dotnet/machinelearning/pull/5202#discussion_r442520500) [](ancestors = 442520500)